sdks/python/apache_beam/ml/inference/base_test.py

```diff
@@ -39,7 +39,7 @@ class FakeInferenceRunner(base.InferenceRunner):
   def __init__(self, clock=None):
     self._mock_clock = clock
 
-  def run_inference(self, batch: Any, model: Any) -> Iterable[Any]:
+  def run_inference(self, batch: Any, model: Any, **kwargs) -> Iterable[Any]:
```

Comment:
I suggest a small unit test just to make sure these parameters get passed through to the inference runner.